package.json

```diff
@@ -42,7 +44,7 @@
     "@vscode/test-cli": "0.0.9",
     "@vscode/test-electron": "2.4.0",
     "eslint": "9.8.0",
-    "typescript": "5.4.5"
+    "typescript": "^5.5.4"
```

Comment:
I'm fine with updating the Typescript version, but it needs to be pinned.

Reply:
That's fine with me.